daml-lf/transaction/src/main/protobuf/com/daml/lf/value.proto

```diff
@@ -36,9 +36,8 @@ message Value {
 
         sint64 int64 = 5;
 
-        // Between v1 and v5 this field expressed a number in base-10 with up
-        // to 28 digits before the decimal point and up to 10 after the decimal
-        // point.
+        // Between v1 and v5 this field expressed a number in base-10 with a
+        // fixed point anywhere within the digits except at the start. 
```

Comment:
Thanks for taking care of thank, however : 

- As far I know this is not use by the JSON API. It is use to persists value in the DB and communcate data between participant. 
- Since #7785 (basically SDK 2.0) we do not support anymore versions 1 to 5, so the comment is now useless. It is probably better to just drop it complelty.